src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -273,10 +273,11 @@ AthensSurfaceExamples class >> example3stroke [
 				can pathTransform rotateByDegrees: i*5.
 				can pathTransform scaleBy: (1- ( i/2000)).
 				can drawShape: path.
-			].
-			Display getCanvas drawImage: surf asForm  at: 0@0 
+			]
 		]	
-	]
+	].
+
+	surf asForm asMorph openInWindow. 
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?